### ISSUE TITLE
Compatibility with Angular 6

### ISSUE THREAD

**Issue by pstjean** [PRX/prx-ng-serve#1]

**Comment by pstjean** [comment]
### Removed
```
inline.bundle.js
vendor.bundle.js
```

### Changed
```
styles.bundle.js --> styles.<hash>.css
main.bundle.js --> main.<hash>.js
scripts.bundle.js --> scripts.<hash>.js
polyfills.bundle.js --> polyfills.<hash>.js
```

### Added
```
runtime.<hash>.js
```

**Comment by cavis** [comment]
Is that for the dev or the prod build?  I think the prod build already had hashes in the filenames.

**Comment by pstjean** [comment]
Ah, you're right, that's comparing dev to prod.

**Comment by pstjean** [comment]
Closed by #2
